alert-policies/temporal/ActivityExecutionFail.yml

```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Activity Execution Fails
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal activity execution fails 1 for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered if the Temporal activity execution fails once within a 5 minutes window.
```

Reply:
Hi @seemantk ,
I made the review comment.